javascript/javascript1/week2/fullName.js

```diff
@@ -0,0 +1,30 @@
+//FLIGHT BOOKING FULLNAME FUNCTIONALITY
+
+console.log('\n***** Homework Javascript Week02 Task 01-FULL NAME" *****');
+
+
+function getFullname(firstname, surname, useFormalName,gender) {
+    if (firstname === '' && surname === '') {
```

Comment:
You can also write it like this for short:
```suggestion
    if (!firstName || !surname) {
```

Reply:
Tak . . . Updated